perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -185,36 +217,31 @@ def score_subreddit(creds_dict,
                                       language=LANGUAGE)
       for e in ensembles:
         scores[e.name] = e.predict_one(scores)
+
+      action_dict = defaultdict(list)
+      for rule in rules:
+        if rule.check_model_rules(scores):
+          action_dict[rule.action_name].append(rule.rule_description)
+          print_moderation_decision(i, comment, rule)
+
+      if mod_permissions and original_mod_permissions:
+        for action, rule_strings in action_dict.items():
+          apply_action(action, comment, rule_strings)
+
       if output_path:
         with open(output_path, 'a') as f:
           record = {
+              'comment_id': comment.id,
               'comment_text': original_comment,
               'scored_comment_text': comment_for_scoring,
               'created_utc': comment.created_utc,
               'permalink': 'https://reddit.com' + comment.permalink,
               'author': comment.author.name,
           }
+          record.update(action_dict)
```

Comment:
does this mean the record has new keys like 'noop' and 'report', with values as lists of actions?

Reply:
Yes. Would you prefer something different or more explicit?

Reply:
I think it's okay, was just ensuring I understood.